src/style.css

```diff
@@ -0,0 +1,158 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  --main-text-color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+.card__container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border: rgb(245, 245, 245) solid 0.5px;
+  border-radius: 5px;
+  width: 200px;
+  height: 408px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-top: 32px;
+  margin-left: 19px;
+  margin-right: 19px;
+}
+
+.card__title {
+  display: flex;
+  margin-top: 40px;
+  margin-left: 16.5px;
+  margin-right: 16px;
+  margin-bottom: 4px;
+  color: var(--main-text-color);
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__code {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  margin-top: 0;
+  margin-left: 16.5px;
+  margin-bottom: 16px;
+  color: #616070;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  margin-left: 1px;
+}
+
+.stars__start {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  width: 13px;
+  height: 13px;
+  margin: 0 7px 0 0;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.stars .stars__start:nth-child(-n+4) {
+  background-image: url("images/star-active.svg");
+}
+
+.card__review {
+  display: flex;
+  flex-direction: row;
+  font-size: 10px;
+  line-height: 14px;
+  margin-left: 17px;
+  color: var(--main-text-color);
+  padding-top: 2px;
+}
+
+.card__price-label {
+  display: flex;
+  color: #616070;
+  text-decoration: none;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+  margin-right: 44px;
+  margin-top: 24px;
+}
+
+.card__price-value {
+  display: flex;
+  color: #060b35;
+  margin-left: 42px;
+  margin-top: 24px;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__buybutton {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  margin: 16px auto;
+  width: 166px;
+  height: 40px;
+  
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: white;
+  background: #00acdc;
+  border-radius: 5px;
+  border: #00acdc solid;
```

Comment:
```suggestion
  border: 1px #00acdc solid;
```

Reply:
sorry, i thought i should press that too